chain-tx-enclave/tx-validation/enclave/src/validate.rs

```diff
@@ -68,12 +68,16 @@ fn construct_sealed_response(
 ) -> Result<IntraEnclaveResponse, sgx_status_t> {
     let to_seal = to_seal_tx.encode();
     match result {
-        Err(e) => Ok(Err(e)),
+        Err(e) => {
+            log::error!("encode tx witoutputs failed: {:?}", e);
+            Ok(Err(e))
+        },
```

Comment:
this is not a failure of encoding -- this will be any time invalid TX is submitted, and the reason for it being rejected is in `e`... no need to log here, as this is returned directly

Reply:
fixed